core/mvcc/database/mod.rs

```diff
@@ -594,10 +663,8 @@ impl<Clock: LogicalClock> MvStore<Clock> {
         tracing::trace!("commit_tx(tx_id={})", tx_id);
         let write_set: Vec<RowID> = tx.write_set.iter().map(|v| *v.value()).collect();
         drop(tx);
-        // Postprocessing: inserting row versions and logging the transaction to persistent storage.
-        // TODO: we should probably save to persistent storage first, and only then update the in-memory structures.
-        let mut log_record = LogRecord::new(end_ts);
-        for ref id in write_set {
+        // Postprocessing: inserting row versions and updating timestamps
+        for ref id in &write_set {
```

Comment:
nit: reference to reference
```suggestion
        for id in &write_set {
```